### PR TITLE
🌐 [i18n-KO] Translated `pad_truncation.mdx` to Korean

### PR DESCRIPTION
**Pull request by sim-so**
# What does this PR do?

Translated the `pad_truncation.mdx` file of the documentation to Korean.
Thank you in advance for your review!

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? 
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/pad_truncation.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+| 모델의 최대 입력 길이로 잘라내기      | 패딩 없음                         | `tokenizer(batch_sentences, truncation=True)` 또는                                        |
+|                                      |                                   | `tokenizer(batch_sentences, truncation=STRATEGY)`                                         |
+|                                      | 배치 내 최대 길이로 패딩          | `tokenizer(batch_sentences, padding=True, truncation=True)` 또는                          |
+|                                      |                                   | `tokenizer(batch_sentences, padding=True, truncation=STRATEGY)`                           |
+|                                      | 모델의 최대 입력 길이로 패딩      | `tokenizer(batch_sentences, padding='max_length', truncation=True)` 또는                  |
+|                                      |                                   | `tokenizer(batch_sentences, padding='max_length', truncation=STRATEGY)`                   |
+|                                      | 특정 길이로 패딩                  | Not possible                                                                              |
```
Comment (sim-so) [inline docs/source/ko/pad_truncation.mdx:57]:
```suggestion
|                                      | 특정 길이로 패딩                  | 사용 불가                                                                              |
```

File: docs/source/ko/pad_truncation.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+| 특정 길이로 잘라내기                 | 패딩 없음                         | `tokenizer(batch_sentences, truncation=True, max_length=42)` 또는                         |
+|                                      |                                   | `tokenizer(batch_sentences, truncation=STRATEGY, max_length=42)`                         |
+|                                      | 배치 내 최대 길이로 패딩          | `tokenizer(batch_sentences, padding=True, truncation=True, max_length=42)` 또는           |
+|                                      |                                   | `tokenizer(batch_sentences, padding=True, truncation=STRATEGY, max_length=42)`           |
+|                                      | 모델의 최대 입력 길이로 패딩       | Not possible                                                                             |
```
Comment (sim-so) [inline docs/source/ko/pad_truncation.mdx:62]:
```suggestion
|                                      | 모델의 최대 입력 길이로 패딩       | 사용 불가                                                                             |
```

File: docs/source/ko/pad_truncation.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  - `True` 또는 `'longest'`: 배치에서 가장 긴 시퀀스로 패딩합니다(단일 시퀀스만 제공하면 패딩이 적용되지 않습니다).
+  - `'max_length'`: `max_length` 인수가 지정한 길이로 패딩하거나, `max_length`가 제공되지 않은 경우(`max_length=None`) 모델에서 허용되는 최대 길이로 패딩합니다. 단일 시퀀스만 제공하는 경우에도 패딩이 적용됩니다.
+  - `False` 또는 `'do_not_pad'`: 패딩이 적용되지 않습니다. 이것이 기본 동작입니다.
+
+`truncation` 인수는 잘라낼 방법을 정합니다. 불리언 또는 문자열일 수 있습니다:
+
+  - `True` 또는 `longest_first`: `max_length` 인수가 지정한 최대 길이로 잘라내거나, 
+    `max_length`가 제공되지 않은 경우(`max_length=None`) 모델에서 허용되는 최대 길이로 잘라냅니다. 
+    이렇게 하면 적절한 길이에 도달할 때까지 시퀀스 쌍에서 가장 긴 시퀀스의 토큰을 제거하며 토큰을 하나씩 잘라냅니다.
```
Comment (sim-so) [inline docs/source/ko/pad_truncation.mdx:29]:
```suggestion
    시퀀스 쌍에서 가장 긴 시퀀스의 토큰을 적절한 길이에 도달할 때까지 하나씩 제거합니다.
```

File: docs/source/ko/pad_truncation.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  - `'only_second'`: `max_length` 인수가 지정한 최대 길이로 잘라내거나, 
+    `max_length`가 제공되지 않은 경우(`max_length=None`) 모델에서 허용되는 최대 길이로 잘라냅니다.
+    이것은 시퀀스 쌍(또는 시퀀스 쌍의 배치)가 제공된 경우 쌍의 두 번째 문장만 잘라냅니다.
```
Comment (sim-so) [inline docs/source/ko/pad_truncation.mdx:32]:
```suggestion
    시퀀스 쌍(또는 시퀀스 쌍의 배치)가 제공된 경우 쌍의 두 번째 문장만 잘라냅니다.
```

File: docs/source/ko/pad_truncation.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  - `'only_first'`: `max_length` 인수가 지정한 최대 길이로 잘라내거나, 
+    `max_length`가 제공되지 않은 경우(`max_length=None`) 모델에서 허용되는 최대 길이로 잘라냅니다. 
+    이것은 시퀀스 쌍(또는 시퀀스 쌍의 배치)가 제공된 경우 쌍의 첫 번째 문장만 잘라냅니다.
```
Comment (sim-so) [inline docs/source/ko/pad_truncation.mdx:35]:
```suggestion
    시퀀스 쌍(또는 시퀀스 쌍의 배치)가 제공된 경우 쌍의 첫 번째 문장만 잘라냅니다.
```

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다 👍
옵션이 적용되는 시퀀스를 명확히 번역해주셔서 쉽게 이해할 수 있었습니다! 
아래 짧은 리뷰를 남깁니다 😀

**Review by wonhyeongseo — APPROVED** [review]
연휴 잘 보내셨나요? 좋은 번역 감사합니다!
LGTM!

**Comment by sim-so** [comment]
Could you review this PR? 😃 
@sgugger, @ArthurZucker, @eunseojo

**Review by ArthurZucker — APPROVED** [review]
Thanks for working on this 😉

**Pull request merged**